cmd/devtools/run.go

```diff
@@ -115,8 +175,56 @@ func runall() {
 	}
 
 	runners := []processrunner.ProcessRunner{seqRunner, cometRunner, compRunner, condRunner}
+	return runners
+}
 
-	// create and start ui app
-	app := ui.NewApp(runners)
-	app.Start()
+func runRemote(ctx context.Context, instanceDir string) []processrunner.ProcessRunner {
```

Comment:
Is there any difference in this and `runLocal` where it couldn't be abstracted into something that loops over a list of `NewProcessRunnerOpts`? May not be super straightforward with the channels and stuff, but would reduce a lot of repetition. Not super important to fix right now, it can wait till later.

Reply:
Decided to wait on fixing this for now.